source/develop/release-management/features/storage/pass_discard_from_guest_to_underlying_storage.html.md

```diff
@@ -0,0 +1,234 @@
+---
+authors: ishaby
+wiki_title: Features/PassDiscardFromGuestToUnderlyingStorage
+feature_name: Pass discard from guest to underlying storage
+feature_modules: engine/vdsm/api-model
+feature_status: Development
+---
+
+# Pass discard from guest to underlying storage
+
+## Summary
+
+### How we use thin provisioned disks today
+Today, when we create a thin provisioned disk on a block domain, a logical volume with a size of 1G is created, regardless its virtual size.<br/>
```

Comment:
This is inaccurate. VDSM exposes the ability to define an initial size. This is used in the v2v and upload image features.